app.js

```diff
@@ -301,6 +282,41 @@ const shouldQuit = app.makeSingleInstance(() => {
   }
 });
 
+const searchPorts = () => {
+  op.find(
+    {
+      // Re: http://www.sikhiwiki.org/index.php/Gurgadi
+      ports: [1397, 1469, 1539, 1552, 1574, 1581, 1606, 1644, 1661, 1665, 1675, 1708],
+      count: 1,
+    },
+    (err, port) => {
+      if (err) {
+        dialog.showErrorBox(
+          'Overlay Error',
+          'No free ports available. Close other applications and Reboot the machine',
+        );
+        app.exit(-1);
+        return;
+      }
+      global.overlayPort = port;
+      // console.log(`Overlay Port No ${port}`);
+      http.listen(port);
+    },
+  );
+};
+
+ipcMain.on('toggle-obs-cast', (event, arg) => {
+  if (arg) {
+    searchPorts();
+  } else {
+    http.close();
```

Comment:
Doesn't look overlay actually turns off. updating lines still updates the webpage.

Reply:
Yes, I have shared about that in `sttm-desktop-dev`, please take a look. 🙏 That's the default behaviour of http.close(). It won't kill existing connections but prevent new connections.